app/controllers/nut_job_authentication_controller.rb

```diff
@@ -0,0 +1,85 @@
+class NutJobAuthenticationController < ApplicationController
+  # skip_before_action(:force_nut_job_sign_in, { :only => [:sign_up_form, :create, :sign_in_form, :create_cookie] })
+
+  def sign_in_form
+    render({ :template => "nut_job_authentication/sign_in.html.erb" })
+  end
+
+  def create_cookie
+    nut_job = NutJob.where({ :email => params.fetch("query_email") }).first
```

Comment:
Usually, we've broken up this step into two: 

First, `params.fetch`

Second, `NutJob.where`

Is the assumption at this point, that they are familiar enough with the process that we can condense it into one line?

Reply:
@jelaniwoods I like it. This reminds me that I feel we've been getting good results with breaking up `.where().at(0)` into two steps also; perhaps we should consider making that canonical, and modifying `draft:resource`.

Reply:
@jelaniwoods However, since this change isn't related to this PR, I am going to defer; can you make an issue to review these concerns?